cmd/gitserver/server/cleanup.go

```diff
@@ -843,14 +850,15 @@ func checkMaybeCorruptRepo(repo api.RepoName, dir GitDir, stderr string) {
 	if !maybeCorruptStderrRe.MatchString(stderr) {
 		return
 	}
+	logger := log.Scoped("checkMaybeCorruptRepo", "check if repo is corrupt")
```

Comment:
```suggestion
	logger := log.Scoped("checkMaybeCorruptRepo", "check if repo is corrupt").With(log.String("repo", string(repo)))
```

Very minor, just a way to avoid passing the repo on the two following log calls :)